src/Box.tsx

```diff
@@ -65,17 +67,22 @@ export const Box = forwardRef<HTMLOrSVGElement, IBox<HTMLOrSVGElement>>((props,
     textTransform,
     color,
     backgroundColor,
+    borderColor,
     transform,
     visibility,
+
+    defaultCSS,
```

Comment:
This is breaking :/

Some 'non UI-Kit' components pass css directly.

Reply:
Those components should be updated to either use the `style` prop or the `defaultCSS` prop. There is too much "magic" involved with the css prop for me to be comfortable using it in such a broad scope especially in non-ui kit components. If the css prop is used there is the issue described above where prop values won't get applied  and that can cause a lot of confusion

Reply:
Main difference between `defaultCSS` is overridable where as `style` is not. So its up to the end user to decide which to apply. But `css` should no longer be used

Reply:
I know there is some Emotion magic behind it, but changing it in all projects will be painful unless we have come up with some codemod for that.
I could write such a codemod if needed (a matter of few mins), but in general I'd like to involve more people to the discussion as IMHO this is quite a significant change and people might be surprised css prop doesn't work anymore.

@marbemac your thoughts?

Reply:
The css prop should still work, we never had logic in Box that handles it. That was handled by the jsx/emotion pragma. The typing was removed so that I guess would be breaking since projects won't compile. So how about we add that back, leave `defaultCSS`, remove `style` (since functionally it is doing the same as `css`)

Reply:
Then once we create better and more organized documentation we make sure to inform everyone the `css` would like html `style` and will override and props? @P0lip

Reply:
yeah, that sounds fine to me Nick

Reply:
I skimmed over this - sounds like ya'll figured it out! Ping me again if a third mind is needed :).